introduction.md

```diff
@@ -4,4 +4,8 @@ Intro
 Here is the intro. Please refer.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?